data/Field.js

```diff
@@ -94,11 +95,19 @@ export const FieldType = Object.freeze({
     STRING: 'string'
 });
 
+/**
+ * @param {string} fieldName - short name / code for a field.
+ * @return {string} - fieldName transformed into user-facing / longer name for display.
+ */
+export function genDisplayName(fieldName) {
+    return fieldName == 'id' ? 'ID' : startCase(fieldName);
```

Comment:
nit: favoring triple equals now

Reply:
I can jump on the `===` bandwagon, no problem.

Reply:
Fixed. We could get serious about this and add it back into the linter if you're ready - probably best to do a bombing there (with some care, `==` has its place) at some point.